### DISCUSSION TITLE
Is it possible to select the text inside noice input?

### DISCUSSION THREAD

**Discussion by Demianeen** [folke/noice.nvim#695] [Q&A] [answered]
Let's say I have an input like this:
![CleanShot 2024-01-16 at 12 19 33@2x](https://github.com/folke/noice.nvim/assets/51330172/5b62b25e-77c1-40e3-8acb-73a628b7c4a7)

In vscode, I could select text with arrow keys + shift. Can I somehow select the text in the input with noice.nvim?

**Comment by ilias777** [comment, marked as answer]
If you have a keybinding to exit insert mode like: `vim.keymap.set('i', 'kj', '<esc>')`, you can enter normal mode with this keybinding and than you can yank the text you want.

https://github.com/folke/noice.nvim/assets/39212564/9dc2874e-5ee2-431d-9230-6f5e4516111d
